src/components/simple-color-picker/simple-color/simple-color.style.ts

```diff
@@ -57,9 +65,18 @@ export const StyledSimpleColorInput = styled(Input)`
   }
 
   &:focus + ${StyledColorSampleBox} {
-    box-shadow: inset 0px 0px 0px var(--borderWidth200)
-      var(--colorsUtilityYang100);
-    border: 2px solid var(--colorsSemanticFocus500);
+    ${({ theme }) => css`
+      ${theme.focusRedesignOptOut &&
```

Comment:
**suggestion** (non-blocking): as mentioned elsewhere I think this could be better as a ternary

Reply:
See previous - this will apply for all ternary suggestions